tests/ignite/contrib/handlers/test_clearml_logger.py

```diff
@@ -331,16 +331,7 @@ def global_step_transform(*args, **kwargs):
 
 def test_weights_scalar_handler_wrong_setup():
 
-    with pytest.raises(TypeError, match="Argument model should be of type torch.nn.Module"):
```

Comment:
Why do you remove these checks ?

Reply:
OK, I got your answer from below comment -> test_base_weights_handler_wrong_setup